byzcoin/service.go

```diff
@@ -78,6 +81,13 @@ func init() {
 	log.ErrFatal(err)
 	network.RegisterMessages(&bcStorage{}, &DataHeader{}, &DataBody{})
 	viewChangeMsgID = network.RegisterMessage(&viewchange.InitReq{})
+
+	if ContractsFn == nil {
```

Comment:
Remove this, and put the `make()` at declaration time, so that it happens one single time, no if checks.